oeps/processes/oep-0055/decisions/0005-managing-core-contributors.rst

```diff
@@ -0,0 +1,71 @@
+0005 Managing Core Contributors Of Your Maintained Repository
+#############################################################
+
+Status
+******
+
+**Under Review**
+
+Context
+*******
+
+Per `OEP-55
+<https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0055-proc-project-maintainers.html>`_,
+repository maintainers are responsible "for the health of the component and the
+component's community". Further, OEP-55 specifies that maintainers must
+participate in the "selection and approval of Core Contributors with commit
+rights to their repositories, by participating in the comment period for Core
+Contributor nominees nominated to commit code to their repo(s)".
+
+However, OEP-55 does not provide sufficient guidance around how Maintainers and
+Core Contributors should interact.
+
+Decision
+********
+
+Maintainers must participate in, and support, the Core Contributor program.
+Maintainers may not take actions that effectively prevent a Core Contributor
+from being able to carry out their Rights, as defined by `OEP-54
+<https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0054-core-contributors.html>`_.
+
+Maintainers should communicate architectural principles and code standards to
+Core Contributors on their repository. They may initiate the process to remove
+Core Contributors that do not follow standards or respond to feedback.
+
+Specification
+*************
+
+Maintainer Responsibilities
+===========================
+
+* Maintainers must participate in the selection of Core Contributors to their
+  repository by expressing their opinion of nominees during the `Core
+  Contributor nomination process <https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0054-core-contributors.html#new-core-contributor-nomination-process>`_
+
+* Maintainers must carefully weigh the merits of a nominee. Maintainers are not
+  allowed to object to every nominee in order to ensure they are the only people
+  with commit access on the repo. For nominees that are objected to, Maintainers
```

Comment:
I agree with your rewording, thank you. I will accept it after some more people look at this PR so they can see your prompt for further discussion. Personally, I don't think it's possible to define an "upper limit" because our repos vary so strongly in terms of scope/focus/complexity, `edx-platform` may always be an outlyer in terms of number of CCs simply because we need a plethora of CCs in order to cover all the different djangoapps and libraries contained within. I could see there being a reasonable case for an upper limit in some cases, for example a repo that had severe security implications on the rest of the platform. But in most cases, no, I don't think defining an upper limit makes sense.